v2/tools/generator/internal/astmodel/package_import.go

```diff
@@ -101,3 +101,9 @@ func (pi PackageImport) VersionedNameForImport() string {
 	service := pi.ServiceNameForImport()
 	return service + pi.packageReference.PackageName()
 }
+
+// WithImportAlias creates a copy of this import with a name following the specified rules
+func (pi PackageImport) WithImportAlias(style PackageImportStyle) PackageImport {
```

Comment:
Do we need the `ServiceNameForImport` and `VersionedNameForImport` above? It feels like they are trying to do something very similar to this?

Reply:
Oh, thanks - I meant to remove those but neglected to. Will remove them now.